src/mixins/enum-mixin.ts

```diff
@@ -239,68 +227,35 @@ export default class EnumMixin extends Vue {
-      case FilingTypes.CHANGE_OF_ADDRESS: return FilingNames.CHANGE_OF_ADDRESS
-      case FilingTypes.CHANGE_OF_COMPANY_INFO: return FilingNames.CHANGE_OF_COMPANY_INFO
-      case FilingTypes.CHANGE_OF_DIRECTORS: return FilingNames.CHANGE_OF_DIRECTORS
-      case FilingTypes.CHANGE_OF_NAME: return FilingNames.CHANGE_OF_NAME
-      case FilingTypes.CHANGE_OF_REGISTRATION: return FilingNames.CHANGE_OF_REGISTRATION
-      case FilingTypes.CONVERSION: return FilingNames.CONVERSION
-      case FilingTypes.CORRECTION: return FilingNames.CORRECTION
-      case FilingTypes.COURT_ORDER: return FilingNames.COURT_ORDER
-      case FilingTypes.DISSOLUTION: return FilingNames.DISSOLUTION
-      case FilingTypes.DISSOLVED: return FilingNames.DISSOLVED
-      case FilingTypes.INCORPORATION_APPLICATION: return FilingNames.INCORPORATION_APPLICATION
-      case FilingTypes.INVOLUNTARY_DISSOLUTION: return FilingNames.INVOLUNTARY_DISSOLUTION
-      case FilingTypes.REGISTRARS_NOTATION: return FilingNames.REGISTRARS_NOTATION
-      case FilingTypes.REGISTRARS_ORDER: return FilingNames.REGISTRARS_ORDER
-      case FilingTypes.REGISTRATION: return FilingNames.REGISTRATION
-      case FilingTypes.RESTORATION: return (subType === RestorationTypes.FULL)
-        ? FilingNames.RESTORATION_FULL
-        : FilingNames.RESTORATION_LIMITED
-      case FilingTypes.SPECIAL_RESOLUTION: return FilingNames.SPECIAL_RESOLUTION
-      case FilingTypes.TRANSITION: return FilingNames.TRANSITION_APPLICATION
-      case FilingTypes.VOLUNTARY_DISSOLUTION: return FilingNames.VOLUNTARY_DISSOLUTION
-      case FilingTypes.PUT_BACK_ON: return FilingNames.PUT_BACK_ON
-    }
-    // fallback for unknown filings
-    return this.camelCaseToWords(type)
+    return EnumUtilities.filingTypeToName(type, agmYear, subType)
   }
 
   /**
+   * DEPRECATED - use enum-utilities instead
    * Converts a string in "camelCase" (or "PascalCase") to separate, title-case words,
    * suitable for a title or proper name.
    * @param s the string to convert
    * @returns the converted string
    */
   camelCaseToWords (s: string): string {
-    return s?.split(/(?=[A-Z])/).join(' ').replace(/^\w/, c => c.toUpperCase()) || ''
+    return EnumUtilities.camelCaseToWords(s)
   }
 
   /**
+   * DEPRECATED - use enum-utilities instead
    * Converts a dissolution type to its name.
    * @param type the dissolution type to convert
    * @returns the dissolution name
    */
   dissolutionTypeToName (type: DissolutionTypes): string {
-    switch (type) {
-      case DissolutionTypes.VOLUNTARY:
-        return this.isFirm ? DissolutionNames.FIRM_DISSOLUTION : DissolutionNames.VOLUNTARY
-      case DissolutionTypes.ADMINISTRATIVE:
-        return DissolutionNames.ADMINISTRATIVE
-    }
-    // fallback for unknown filings
-    return this.camelCaseToWords(type)
+    return EnumUtilities.dissolutionTypeToName(this.isFirm, type)
   }
```

Comment:
As we discussed with @severinbeauvais, mixins are not supported in Vue3 and, more importantly, I can't call them from getters.  I've moved the logic to a new class using all static methods and converted this mixin to an adapter.